index.html

```diff
@@ -117,56 +131,62 @@ <h2>About me</h2>
       </p>
       <button>Get my Resume</button>  
     </section>
-    <div class="languages">
-      <div class="image"><a href=""><img src="images/Languages.svg" alt="languages"></a></div>
-      <h2>Languages</h2>
-      <ul>
-        <li>Ruby on rails</li>
-        <li>css</li>
-        <li>JavaScript</li>
-        <li>html</li>
-      </ul>
-    </div>
-    <div class="framework">
-      <div class="image"><a href=""><img src="images/Frameworks.svg" alt="Framework"></a></div>
-      <h2>Framework</h2>
-      <ul>
-        <li class="framitem1">
+    <!--desktop grid-->
+    <section class="grid-skills">
+      <div class="grid-skills-items">
+        <!--mobile version-->
+        <div class="languages">
+          <div class="image"><a href=""><img src="images/Languages.svg" alt="languages"></a></div>
+          <h2>Languages</h2>
           <ul>
-            <li>Bootstrap</li>
             <li>Ruby on rails</li>
-            <li>Rspec</li>
-          </ul>
-        </li>
-        <li class="framitem2"> 
-          <ul>
-            <li>Capybara</li>
-            <li>Selenium</li>
+            <li>css</li>
+            <li>JavaScript</li>
+            <li>html</li>
           </ul>
```

Comment:
- [ ] Starting from the `about me` section to the end of the page Please fix your styling to be in the center because it's supposed to be in the middle.
- [ ] Kindly fix the background positioning for the form section and try to stick as much as possible to the given Figma design.

| Figma Design  | Your Design |
| ------------- | ------------- |
| ![image](https://user-images.githubusercontent.com/68796232/161077827-45c05535-6515-49b5-b16a-d7171d06b0c9.png)  | ![image](https://user-images.githubusercontent.com/68796232/161077439-080fc0f2-6e23-4356-abd7-09a841a6c5c1.png)|
| ![image](https://user-images.githubusercontent.com/68796232/161077985-9e5e43c5-e193-4940-b93f-8eda9255ecbd.png) | ![image](https://user-images.githubusercontent.com/68796232/161077511-16590a0f-394e-437a-8a22-01d2c08d1311.png) |